MYREADME.md

```diff
@@ -0,0 +1,37 @@
+# Chitter Challenge
```

Comment:
Nice database setup tutorials. Include a 'getting started' section. 
Screenshots of website.